cas-server-documentation/installation/User-Interface-Customization.md

```diff
@@ -364,15 +364,19 @@ associated theme to selectively choose which set of UI views will be used to gen
 for a different type of audience are entirely different structurally that simply
 using the `ServiceThemeResolver` is not practical to augment the default views. In such cases, new view pages may be required.
 
-Views associated with a particular theme by default are expected to be found at: `/WEB-INF/view/jsp/<theme-id>/ui/` 
+Replace the `internalViewResolver` definition with the following:
 
 {% highlight xml %}
-<bean id="themeResolver" class="org.jasig.cas.services.web.RegisteredServiceThemeBasedViewResolver"
-    c:defaultThemeName="${cas.themeResolver.defaultThemeName}"
+<bean id="internalViewResolver" class="org.jasig.cas.services.web.RegisteredServiceThemeBasedViewResolver"
```

Comment:
Why change the bean name?

See http://docs.spring.io/spring/docs/current/javadoc-api/org/springframework/web/servlet/DispatcherServlet.html